data/aws_cluster.json

```diff
@@ -1,11 +1,10 @@
 {
   "num_workers": 1,
-  "cluster_name": "API_Metastore_Work_Leave_Me_Alone",
+  "cluster_name": "Workspace_Migration_Work_Leave_Me_Alone",
```

Comment:
Because this file is used whenever `launch_cluster()` is called (not just for Metastore work), we rename the cluster_name to be more generic